ui/js/controllers/bugfiler.js

```diff
@@ -73,13 +91,22 @@ treeherder.controller('BugFilerCtrl', [
                 }
             }
 
-            $uibModalInstance.possibleFilename = summary[0].split("/").pop();
-
+            // Try unix paths first, then Windows, then bail
+            var splitFilePath;
+            if(summary[0].split("/").length > 1) {
```

Comment:
Things like this would be great in another function, to make this one shorter and more readable